frontend/src/pages/Browse/Browse.tsx

```diff
@@ -1,11 +1,30 @@
-import { Card } from "@/components/Card";
+import { ProductColumn } from "@/components/ProductColumn";
 import { Input } from "@/components/retroui";
 
 import nwData from "../../../../backend/app/assets/data/nw_actual.json";
 import pnsData from "../../../../backend/app/assets/data/pns_actual.json";
 import wlsData from "../../../../backend/app/assets/data/wls.json";
 
 export function Browse() {
+  if (!nwData && !pnsData && !wlsData) {
```

Comment:
This makes it so we can pass in undefined for the tests, checks that the props exist ad not just defaut to the imports

```suggestion
if (!nwProp && !pnsProp && !wlsProp) {
```